tests/test_middleware.py

```diff
@@ -146,4 +146,4 @@ def test_middleware_order():
     request, response = app.test_client.get('/')
 
     assert response.status == 200
-    assert order == [1,2,3,4,5,6]
```

Comment:
Are all these changes to other tests necessary for this PR?

Reply:
Actually no :/
My editor flake checker noticed me some problems and I fixed them, If you think that they should remain virgin, I'll respect that.

Reply:
If we could leave those out of this PR and put them in as a separate PR that would be awesome!

Reply:
I fixed that, I will send a PR to solve flake problems.
If there are other this kind of problems in my code, warn me to don't repeat it in next pull requests!